gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/equality/EqualityTest.java

```diff
@@ -286,10 +297,11 @@ public void testUndirectedGraphEquality() throws Exception {
     FlinkAsciiGraphLoader loader =
       getTestGraphLoader();
 
-    GraphEquality equality = new GraphEquality(
-      new GraphHeadToDataString(),
-      new VertexToDataString(),
-      new EdgeToDataString(),
+    GraphEquality<EPGMGraphHead, EPGMVertex, EPGMEdge, LogicalGraph, GraphCollection> equality =
+      new GraphEquality<>(
```

Comment:
Indentation: arguments to GraphEquality constructor should have more indentation.